samples/videoDecodeRGB/videodecrgb.cpp

```diff
@@ -374,6 +402,9 @@ int main(int argc, char **argv) {
             int last_index = 0;
             for (int i = 0; i < n_frames_returned; i++) {
                 p_frame = viddec.GetFrame(&pts);
+                if (b_generate_md5) {
+                    viddec.UpdateMd5ForFrame(p_frame, surf_info);
+                }
```

Comment:
Do you intend to calculate MD5 only on decoded YUV content? If so, there is no need to add MD5 to this sample since it is redundant to videoDecode sample. If we plan to calculate MD5 on the converted RGB content, we need to call update MD5 after YUV to RGB conversion. AND, the UpdateMd5ForFrame() function can not be used since it is written for the decoded YUV planes from VCN only. Now the RGB buffer is packed format and in host memory already. We need to write a new MD5 update function for RGB frame.